apps/glusterfs/errors.go

```diff
@@ -10,26 +10,26 @@
 package glusterfs
 
 import (
-	"errors"
+	stdErrors "errors"
```

Comment:
I don't understand this change. There's no import of the enhanced errors package here.

Reply:
I like to make it explicit here, that this is the only place where still the "old" errors package is used. Also it avoids, that when code is added to the file (like I did in follow up PRs) doesn't accidentally use the standard errors package.